pkg/domain/aggregates/cluster.go

```diff
@@ -142,26 +130,46 @@ func (a *ClusterAggregate) ApplyEvent(event es.Event) error {
 		if err != nil {
 			return err
 		}
-		a.displayName = clusterCreatedV2.GetDisplayName()
 		a.name = clusterCreatedV2.GetName()
 		a.apiServerAddr = clusterCreatedV2.GetApiServerAddress()
 		a.caCertBundle = clusterCreatedV2.GetCaCertificateBundle()
+	case events.ClusterCreatedV3:
+		clusterCreatedV3 := new(eventdata.ClusterCreatedV3)
+		err := event.Data().ToProto(clusterCreatedV3)
+		if err != nil {
+			return err
+		}
+		a.name = clusterCreatedV3.GetName()
+		a.apiServerAddr = clusterCreatedV3.GetApiServerAddress()
+		a.caCertBundle = clusterCreatedV3.GetCaCertificateBundle()
 	case events.ClusterUpdated:
 		data := new(eventdata.ClusterUpdated)
 		err := event.Data().ToProto(data)
 		if err != nil {
 			return err
 		}
 
-		if len(data.GetDisplayName()) > 0 && a.displayName != data.GetDisplayName() {
```

Comment:
why remove it? `ClusterUpdated(V1)` still has the display name attribute 🤔

Reply:
Cause it has been removed from the projection as well. So it doesn't make sense to process it anymore for old commands.